core/integration/features/spot/accounts_and_fees/trading_fees_spot.feature

```diff
@@ -1,6 +1,8 @@
 Feature: Spot market
 
-  Scenario: 001 when an order is placed, holding accout should have the holding asset; when there is not enough asset to move to holding account, the order can not be placed. When trade happens, holding asset is released: 0080-SP-FEES-001; 0080-SP-FEES-002; 0080-SP-FEES-003; 0080-SP-FEES-004; 0080-SP-FEES-005; 0080-SP-FEES-006; 0080-SP-FEES-007
+  Scenario: 001 when an order is placed, holding account should have the holding asset; when there is not enough asset to move to holding account,
+            the order can not be placed. When trade happens, holding asset is released: 0080-SP-FEES-001; 0080-SP-FEES-002; 0080-SP-FEES-003;
+            0080-SP-FEES-004; 0080-SP-FEES-005; 0080-SP-FEES-006; 0080-SP-FEES-007
```

Comment:
this looks broken, isn't it?

Reply:
Those references were there before I touched it, not sure what they are linked to so I didn't touch them (other than formatting)